README.rst

```diff
@@ -33,7 +33,7 @@ true``, more information `here
 
 If you want to try it locally, a Kerberos token can be obtained via::
 
-   $ docker run -i -t --rm reanahub/reana-auth-krb5:1.0.1 /bin/bash
+   $ docker run -i -t --rm reanahub/reana-auth-krb5:1.0.2 /bin/bash
```

Comment:
I would suggest to use 1.1.0 instead, since renewing tokens is a significant change in the functionality.